CodeStyle.Analyzers.Test/CodeStyle.Analyzers.Test.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
+    <RootNamespace>Messerli.CodeStyle.Analyzers.Test</RootNamespace>
```

Comment:
We usually also set the assembly name

Reply:
[Set assembly name](https://github.com/messerli-informatik-ag/code-style/pull/63/commits/30e1939cdf072926b44cd415608a1cde4b4dfb7d)